spec/models/user_spec.rb

```diff
@@ -1,5 +1,28 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'admin?' do
+    context 'neighborhood user is an admin' do
+      it 'returns true' do
+        user = FactoryGirl.create(:user, role: 'admin')
+        expect(user.admin?).to eq true
```

Comment:
here's some neat rspec syntax:

```
expect(user).to be_admin
```

It's RSpec magic. It is smart about any method that ends with a `?`